shared/build.gradle.kts

```diff
@@ -58,7 +58,11 @@ kotlin {
                 implementation("junit:junit:4.13.2")
             }
         }
-        val iosMain by getting
+        val iosMain by getting {
+            dependencies {
+                implementation("io.ktor:ktor-client-ios:1.4.0")
```

Comment:
Cool, didn't know that the ktor client is needed as a dependency for ios as well.